nautobot/core/ui/object_detail.py

```diff
@@ -683,12 +695,29 @@ def render_body_content(self, context):
             return format_html('<tr><td colspan="2">{}</td></tr>', placeholder(data))
 
         result = format_html("")
+        panel_label = slugify(self.label or "")
         for key, value in data.items():
             key_display = self.render_key(key, value, context)
-            value_display = self.render_value(key, value, context)
-            if value_display:
-                # TODO: add a copy button on hover to all display items
-                result += format_html("<tr><td>{key}</td><td>{value}</td></tr>", key=key_display, value=value_display)
+            if value_display := self.render_value(key, value, context):
+                if value in [None, ""]:
```

Comment:
What about the case where `value` is an empty queryset?

Maybe it might be simpler/cleaner to compare `value_display` against the `HTML_NONE` constant instead?